src/aind_metadata_service/server.py

```diff
@@ -39,7 +39,7 @@
 
 
 @app.get("/subject/{subject_id}")
-async def retrieve_subject(subject_id):
+async def retrieve_subject(subject_id, pickle=False):
```

Comment:
Maybe add a comment or a note in the docstring that the pickle parameter will be handled in a future version

Reply:
Also add a note here in the server.py file